app/react-native/package.json

```diff
@@ -27,6 +27,7 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
+    "@react-native-community/async-storage": "^1.2.1",
```

Comment:
let's bring the dev/peer react-native dependency to 57 as part of this. 57 is 6 months old at this point, should be ok to make it a requirement for storybook 5.x

Reply:
Will do!

Reply:
I just updated the version. Hopefully CI is ok with it, when [the codebase I primarily work on](https://github.com/skyscanner/backpack-react-native) went from 55 to 57, upgrading was really difficult.